mbuild/formats/charmm_writer.py

```diff
@@ -1212,14 +1254,22 @@ def __init__(
             its original order, as in the Compound sent to the writer.
             If True, the order of the atoms is reordered based on their
             residue names in the 'residues' list that was entered.
-        box_0 : list, [x-dim, y-dim ,z-dim]
-            A list of 3 positive float values or the dimensions [x, y ,z]
-            for structure_box_0 in nanometers (nm)
-            This is to add/override or change the structures dimensions. Ex: [1,2,3]
-        box_1 : list, [x-dim, y-dim ,z-dim]
-            A list of 3 positive float values or the dimensions [x, y ,z]
-            for structure_box_1 in nanometers (nm)
-            This is to add/override or change the structures dimensions. Ex: [1,2,3]
+        box_0 : Box
+            The Box class that contains the attributes Lx, Ly, Lz for the length
+            of the box 1 in nm. It also contains the xy, xz, and yz Tilt factors
```

Comment:
I think you mean `box 0` here?

Also, I want to confirm about the unit difference between these variables:
- `box_0`: `nm`
- `box_0_vectors`: `Ang`
- `box_1`: `Ang`
- `box_1_vectors`: `Ang`

Reply:
on the portion of code below, I think `box_1` is also in `nm` (conversion starting from line 1952), so I assume this is just a typo in the docstring. but I still need confirmation on the unit of the vectors

Reply:
Good catch!!  Thanks.  Yeah it should be the following with the box_X being nm for standard mbuild units and box_x_vectors for the GOMC units (Ang):

- box_0: nm
- box_0_vectors: Ang
- box_1: nm
- box_1_vectors: Ang